components/resources/jobExplorer/jobList/JobListComponent.tsx

```diff
@@ -1,14 +1,12 @@
 import Image from "next/legacy/image";
 import React from "react";
 
-
-const disabledJobs = [
```

Comment:
looks like my formatter settings are a bit different from others. we should all align on using the same vscode formatter settings to avoid these unecessary changes in PRs.